src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__photo" src="/images/imac.jpeg" alt="Моноблок APPLE A1419 iMac 27&#34 Retina 5K">
```

Comment:
Краще розділити на різні рядки
```suggestion
      <img 
        class="card__photo" 
        src="/images/imac.jpeg" 
        alt="Моноблок APPLE A1419 iMac 27&#34 Retina 5K"
      >
```